storage/demo/src/main/res/layout/fragment_scan.xml

```diff
@@ -9,13 +9,24 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:minHeight="?attr/actionBarSize"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:menu="@menu/fragment_scan"
+        app:navigationIcon="?attr/homeAsUpIndicator" />
+
     <androidx.core.widget.NestedScrollView
         android:id="@+id/scrollView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="0dp"
```

Comment:
do you know why this works? There's only a top constraint and no bottom. So why doesn't this shrink it to actual height `0`?

Reply:
@grote not sure. Do you want me to put a bottom constraint here though?

Reply:
It *does* seem to work, so this shouldn't be needed, I guest wonder how this can even work :shrug: